docs/chat/chat.yaml

```diff
@@ -62,22 +62,62 @@ paths:
               schema:
                 $ref: '#/definitions/chat'
               example:
-                - _id: 64a54c6c7bca605aaf2da66c
+                - _id: 67604def753afa7bbd3bd98a
                   members:
-                    - user: 6491d003a634c3c427b69daa
+                    - user:
+                        _id: 67519181170a70623366d30c
+                        firstName: Diana
+                        lastName: Test
                       role: tutor
-                    - user: 6421d9833cdf38b706756dff
+                    - user:
+                        _id: 673f342709e56a9071f5308c
+                        firstName: valya
+                        lastName: dudchak
+                        professionalSummary: ""
+                        photo: 1733337385040-photofordemo.jpg
```

Comment:
Why user have different fields in each example?

Reply:
the first user does not have their photo uploaded, as well as professionalSummary. so it does not return in the service